README.md

```diff
@@ -89,6 +89,16 @@ var opts = {
 }
 
 logger.log('My Sample Log Line', opts);
+
+// or pass any associated  object or a string along all lines of a logger
+// note: it will parse only two levels of the object
+logger.addProperty({'loan':'signing'});
+
+// these lines will be associated with the passed object:
+logger.log('Sending the loan for signing...');
+logger.log('The user has received the request to sign');
+
+logger.removeProperty({'loan':'signing'});
```

Comment:
There is something wrong here - first of all, `removeProperty` doesn't take any input; secondly, `removeProperty` should take list of keys to remove from the ambient meta part... Let's not include the `Ambient Meta` details into the docs yet.

Reply:
uh right... it does not take any properties. Hm... yes we can also add a functionality to remove only specific keys from the ambient property in the case if the ambient meta is an object...